```
      Restored /__w/1/vmr/artifacts/scenario-tests/artifacts/SdkTemplateTestsAot_Web_CSharp/SdkTemplateTestsAot_Web_CSharp.csproj (in 2.08 sec).
    /__w/1/vmr/artifacts/obj/extracted-dotnet-sdk/sdk/10.0.100-alpha.1.25072.1/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.RuntimeIdentifierInference.targets(326,5): message NETSDK1057: You are using a preview version of .NET. See: https://aka.ms/dotnet-support-policy [/__w/1/vmr/artifacts/scenario-tests/artifacts/SdkTemplateTestsAot_Web_CSharp/SdkTemplateTestsAot_Web_CSharp.csproj]
      SdkTemplateTestsAot_Web_CSharp -> /__w/1/vmr/artifacts/scenario-tests/artifacts/SdkTemplateTestsAot_Web_CSharp/bin/Release/net10.0/linux-x64/SdkTemplateTestsAot_Web_CSharp.dll
/__w/1/vmr/src/scenario-tests/.packages/microsoft.dotnet.ilcompiler/10.0.0-alpha.1.25071.14/build/Microsoft.NETCore.Native.Publish.targets(71,5): error : The PrivateSdkAssemblies ItemGroup is required for _ComputeAssembliesToCompileToNative [/__w/1/vmr/artifacts/scenario-tests/artifacts/SdkTemplateTestsAot_Web_CSharp/SdkTemplateTestsAot_Web_CSharp.csproj] [/__w/1/vmr/src/scenario-tests/src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/Microsoft.DotNet.ScenarioTests.SdkTemplateTests.csproj] [/__w/1/vmr/repo-projects/scenario-tests.proj]
##[error]src/scenario-tests/.packages/microsoft.dotnet.ilcompiler/10.0.0-alpha.1.25071.14/build/Microsoft.NETCore.Native.Publish.targets(71,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) The PrivateSdkAssemblies ItemGroup is required for _ComputeAssembliesToCompileToNative [/__w/1/vmr/artifacts/scenario-tests/artifacts/SdkTemplateTestsAot_Web_CSharp/SdkTemplateTestsAot_Web_CSharp.csproj]
```

@sbomer are you the right person to ask for help here regarding the `Microsoft.NETCore.Native.Publish.targets` error? The source of the failing test is here: https://github.com/dotnet/scenario-tests/blob/7790d479e5f6bd16dc132257d49344b1c7c79c5f/src/Microsoft.DotNet.ScenarioTests.SdkTemplateTests/SdkTemplateTests.cs#L222-L231

--- a/src/SourceBuild/content/repo-projects/scenario-tests.proj
+++ b/src/SourceBuild/content/repo-projects/scenario-tests.proj
@@ -8,6 +8,8 @@
     <ScenarioTestsResultsDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsTestResultsDir)', 'scenario-tests'))</ScenarioTestsResultsDir>
     <!-- Use the updated NuGet.config file from the intermediate folder which contains the live local feeds. -->
     <NuGetConfigInputForScenarioTests>$([MSBuild]::NormalizePath('$(ArtifactsObjDir)', 'sdk', 'NuGet.config'))</NuGetConfigInputForScenarioTests>
+    <!-- TODO: Consolidate with the above line when building source-only: https://github.com/dotnet/source-build/issues/4841 -->
+    <NuGetConfigInputForScenarioTests Condition="'$(DotNetBuildSourceOnly)' == 'true'">$([MSBuild]::NormalizePath('$(SrcDir)', 'sdk', 'NuGet.config'))</NuGetConfigInputForScenarioTests>
     <NuGetConfigOutputForScenarioTests>$(ScenarioTestsArtifactsDir)NuGet.config</NuGetConfigOutputForScenarioTests>
   </PropertyGroup>
 

